source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Autodefine External Network
+category: feature
+authors: Ales Musil
+feature_name: Autodefine External Network
+feature_modules: engine
+feature_status: In Development
+---
+
+# Autodefine External Network
+
+## Summary
+
+This feature extends [Provider Physical Network](https://www.ovirt.org/develop/release-management/features/network/provider-physical-network/) 
+adding support for auto-defined external network connected to physical ones. 
+With this feature implemented, when a physical VM network is created, Engine 
+automatically creates an external network based on it, and connects the external
+network to the physical one. This makes it easier to use the former feature, and
+helps oVirt to model host networks and VM networks separately.
+
+Related patches can be found on [gerrit topic:autodefine](https://gerrit.ovirt.org/#/q/topic:autodefine).
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+### Benefit to oVirt
+
+Provider Physical Network is useful for users who would like to 
+manage their network via an SDN (such as OVN or Neutron) yet have near-native 
+performance. However, without auto-define feature, such admin would have to 
+create a physical network (and attach it to hosts), create an external network,
+and remember to connect this two networks together. Autodefine helps reducing 
+these administrative tasks. In an OvS cluster with an external provider 
```

Comment:
I think that term "OvS" should be either "OvS" or "OVS" along the entire document

Reply:
Done